exercises/practice/sgf-parsing/.meta/example.ex

```diff
@@ -175,19 +168,15 @@ defmodule SgfParsing do
     end
   end
 
-  def drop_and(p1, p2) do
+  defp drop_and(p1, p2) do
     fn string ->
       with {:ok, _, rest} <- p1.(string) do
         p2.(rest)
       end
     end
   end
 
-  def inject(a) do
```

Comment:
`string`, `inject` - also dead code?

Reply:
Yup. I was very surprised to realize I didn't need them in the end, but I didn't have the heart to delete them. Bad Jie!

Reply:
Sounds like somebody didn't pay too much attention to the example solutions when doing the code review for the new exercises 😁